docs/source/integration.rst

```diff
@@ -0,0 +1,94 @@
+==========================================
+Integrating other Django Apps into Glitter
+==========================================
+
+Glitter App Pages
+=================
+
+Pages for standard Django Apps can be added to Glitter's page hierarchy easily by using the Glitter
+App Pages feature, which allows:
+
+* standard Django apps to appear in the navigation alongside Glitter Pages
+* admin users to be able to set the URL & menu position for standard Glitter apps
+* integration with page selection widgets and other bits & bobs which rely on the Glitter Page
+  structure.
+
+Instructions
+------------
+
+1) Add/create/magic-up a standard Django app to your project as you normally would
+
+2) Ensure the Django app has a root (`r'^$',`) route in its `urls.py`. For example, the Glitter
+   News app has the following root route in `glitter_news/urls.py`::
+
+       urlpatterns = [
+           url(
+               r'^$',
+               views.PostListView.as_view(),
+               name='list'
+           ),
+
+           ...
+        ]
+
+3) Setup your project's main `urls.py` to support Glitter App Pages by removing any existing URLs
+   you might have for the Django apps you're integrating into Glitter. If you don't remove these,
+   then they tend to conflict with the Glitter App Pages and you'll get unpredictable results.
+
+4) Add the `glitter.pages.middleware.GlitterUrlConfMiddleware` to your project's settings. This
+   should be added before Glitter's other middleware. Something like this will work::
+
+       MIDDLEWARE_CLASSES = [
+           'django.contrib.sessions.middleware.SessionMiddleware',
+           'django.middleware.common.CommonMiddleware',
+           'django.middleware.csrf.CsrfViewMiddleware',
+           'django.contrib.auth.middleware.AuthenticationMiddleware',
```

Comment:
Something like this is more readable, don't think we need to tell to the user what `Django` middlewares should be in used 
```      
MIDDLEWARE_CLASSES = [
           ...
           'glitter.pages.middleware.GlitterUrlConfMiddleware',  # Before other Glitter middleware
           'glitter.pages.middleware.PageFallbackMiddleware',
           'glitter.middleware.ExceptionMiddleware',
]
```